contrib/clojure-package/examples/infer/objectdetector/src/infer/objectdetector_example.clj

```diff
@@ -109,7 +114,7 @@
     (apply concat
```

Comment:
Why do we need a concat here? Cant we use a `doseq` instead of `for` to force the lazy computation?

Reply:
Good idea, but it's also used in the example test to verify the results.